tests/cypress/support/commands.js

```diff
@@ -214,7 +214,13 @@ Cypress.Commands.add(
                 errorMsg: 'Timeout waiting for all layers to be loaded',
             }
         )
-        if ('3d' in queryParams && queryParams['3d'] === true) {
+
+        // In the legacy URL, 3d is not found. We check is the map in 3d or not by checking the pitch, heading, and elevation
```

Comment:
another minor typo here `We check if the map...` and not `We check is the map...`